### ISSUE TITLE
New translation string(s) ?

### ISSUE THREAD

**Issue by trebmuh** [liushuyu/lmms-translations#2]
@liushuyu : hi, is there any translation string ?

**Comment by liushuyu** [comment]
> @liushuyu : hi, is there any translation string ?

Unfortunately, yes 😄. And I will update more strings today.

**Comment by trebmuh** [comment]
No worries. It would be great if you can sync the files here then.

**Comment by liushuyu** [comment]
Sorry for being late, but time is tight for a university student...

The files are refreshed, please note that the `FDBK` in new translation means `Feedback`

**Comment by liushuyu** [comment]
@trebmuh BTW, I have written a small script for you to easily validate your translations: https://github.com/liushuyu/DraftBin/tree/master/pyscripts/ts-verify
You need to download the py file along with the xsd file and place them in the same directory.

You will need to install `lxml`(python module) and `python3` in order to get this scriptlet run.

```
# apt-get install python3 python3-pip python3-lxml
```

**Comment by liushuyu** [comment]
@trebmuh ping?

**Comment by liushuyu** [comment]
@trebmuh So how are you doing? The newer translation strings are up...

**Comment by trebmuh** [comment]
Hell, I totally forgot about this one!
I'll try to make a fresh update in the following weeks.

**Comment by liushuyu** [comment]
That's okay, I am glad you are still around. Take your time. Also, thanks for your brilliant contributions.

**Comment by trebmuh** [comment]
@liushuyu : it wasn't a big deal (not a lot of new strings), so it's already updated. Please, see https://github.com/liushuyu/lmms-translations/pull/3

**Comment by trebmuh** [comment]
Note that I didn't try your validating script. I'll look into that later when I've got time to do so. In the meantime, please let me know if this PR breaks the compilation.

**Comment by liushuyu** [comment]
>Note that I didn't try your validating script. I'll look into that later when I've got time to do so. In the meantime, please let me know if this PR breaks the compilation.

That's okay, if you don't want to try that script, I will validate for you. 

And btw, I remembered you are using Debian, right? I suppose you could install `qttools5-dev-tools`, so that you can edit translation files in `linguist` program, which is easier.

**Comment by trebmuh** [comment]
Awesome. Yes, I know about the translation tools, but I'm much quicker with text editor. I might be a bit old school here, I know :)

**Comment by liushuyu** [comment]
>Awesome. Yes, I know about the translation tools, but I'm much quicker with text editor. I might be a bit old school here, I know :)

Yes, in this case, we respect your preference.

**Comment by liushuyu** [comment]
@trebmuh Hi, the latest strings are now up, do you still have time to work on this ?

**Comment by trebmuh** [comment]
@liushuyu : done here https://github.com/liushuyu/lmms-translations/pull/4
I didn't rebuild to check that everything is working properly, but I've double-check the pull request to ensure that there is no obvious code mistake. It should be safe.
Thanks for the ping !

**Comment by liushuyu** [comment]
@trebmuh Hi, so LMMS development is very active these days, someone add a new plugin named ReverbSC, there are new strings from this new plugin (https://github.com/LMMS/lmms/pull/3202). Do you have sometime to work on new translations? Thanks.

**Comment by trebmuh** [comment]
@liushuyu : thanks for the ping, done : https://github.com/liushuyu/lmms-translations/pull/5

**Comment by trebmuh** [comment]
@liushuyu : hi, no new translation strings?

**Comment by liushuyu** [comment]
>@liushuyu : hi, no new translation strings?

Sorry for late response, LMMS is undergoing some changes, I will collect all the source strings asap. Thanks.

**Comment by trebmuh** [comment]
All good, please ping me when this is ready @liushuyu

**Comment by liushuyu** [comment]
>All good, please ping me when this is ready @liushuyu

Sorry for the long wait, but the source strings here have been updated.

**Comment by trebmuh** [comment]
Hi @liushuyu thanks and done : https://github.com/liushuyu/lmms-translations/pull/8

**Comment by trebmuh** [comment]
@liushuyu : hi
looking at https://github.com/LMMS/lmms/blob/1.2.0-rc3/data/locale/fr.ts , I'm wondering if everything (French translation) will be merged before the official 1.2 release.
I'd like to be able to check if everything is correct before the last run.

**Comment by liushuyu** [comment]
@trebmuh Hi, there are some new translation strings available.

>@liushuyu : hi
looking at https://github.com/LMMS/lmms/blob/1.2.0-rc3/data/locale/fr.ts , I'm wondering if everything (French translation) will be merged before the official 1.2 release.
I'd like to be able to check if everything is correct before the last run.

Your translations will of course be merged in 1.2 release, we are just waiting for other translators from other languages to finish their jobs (or at least partially translated).

**Comment by trebmuh** [comment]
@liushuyu : done: https://github.com/liushuyu/lmms-translations/pull/9
>Your translations will of course be merged in 1.2 release, we are just waiting for other translators from other languages to finish their jobs (or at least partially translated).

Wonderfull! I'll be nice to have a 1.2 version release including a full French translation for the French readers ! Love it.

**Comment by liushuyu** [comment]
@trebmuh Are you interested in translating our __new__ website (Preview here: http://lmms.io:8008) ? 
The source files are located in `lmms.io/`

**Comment by trebmuh** [comment]
@liushuyu : yes.

I've looked at it quickly and, before starting reviewing what's already done and translating what's not yet, I'd like to ask you for something : I can see in some comments, some references to paths whose seems to be on your local computer, e.g. https://github.com/liushuyu/lmms-translations/blob/master/lmms.io/fr.po#L22 .

Is that normal/expected ?

**Comment by trebmuh** [comment]
@liushuyu ping

**Comment by trebmuh** [comment]
@liushuyu : no news from you? I hope everything is going well. Please ping back if you can.

**Comment by trebmuh** [comment]
@liushuyu : ?

**Comment by liushuyu** [comment]
Hmm… sorry for the late response, I am currently having heavy loads of school work. 

And I'll update the strings as soon as possible

**Comment by trebmuh** [comment]
@liushuyu no worries, I understand, we're all there.
Please ping me when you've updated the strings and I'll act towards it.

**Comment by liushuyu** [comment]
>Please ping me when you've updated the strings and I'll act towards it.

@trebmuh , Okay, all translation strings are up-to-date now

**Comment by liushuyu** [comment]
And note for the website translation: once the translation reaches 100%, it will be available at http://lmms.io:8008

**Comment by trebmuh** [comment]
Awesome, thanks.
I'm on it, I'll PR it once I'm done.

**Comment by trebmuh** [comment]
@liushuyu : done https://github.com/liushuyu/lmms-translations/pull/10

**Comment by trebmuh** [comment]
I'm closing this github issue here since it diverges from the beginning. We can open others if/when needed.
